### ISSUE TITLE
[Impeller] compute minimum input hint for shared backdrop layer.

### ISSUE THREAD

**Issue by jonahwilliams** [flutter/flutter#157110]
Right now we're defaulting to blurring the entire backdrop, but this can be speed up by computing the union of all saveLayer coverages during the pre-pass.

**Comment by gaaclarke** [comment]
I'll take a stab at this.

**Comment by gaaclarke** [comment]
This was attempted in https://github.com/flutter/engine/pull/56006 but it was discovered that we'd have to expand the bookkeeping of the first pass and it was decided that we should get more data about performance before investing in that extra bookkeeping cost and complexity.

**Comment by jonahwilliams** [comment]
yeah I was wrong :) , we do need to compute the clipRect because that is what shrinks the size of backdrop filters
